nais/dev-sbs/personbruker.json

```diff
@@ -1,6 +1,7 @@
 {
   "namespace": "personbruker",
   "ingresses": [
-    "https://www.dev.nav.no/person/dittnav"
+    "https://www.dev.nav.no/person/dittnav",
+    "https://dittnav-personbruker.nais.oera-q.local"
```

Comment:
Denne burde være `https://dittnav.personbruker.nais.oera-q.local`, siden konvensjonene skal være <appnavn>.<namespace>.<felles for alle team>

Reply:
Ok, har fikset det